servers/rendering/renderer_rd/renderer_scene_render_rd.cpp

```diff
@@ -732,6 +735,11 @@ void RendererSceneRenderRD::_render_buffers_debug_draw(const RenderDataRD *p_ren
 		}
 	}
 
+	if (debug_draw == RS::VIEWPORT_DEBUG_DRAW_INTERNAL_BUFFER) {
```

Comment:
Should probably have made this remark earlier on, but we need a better name then internal buffer. Internal buffer is far to non-descript. Which internal buffer? We have dozens of internal buffers...

Reply:
Do you think we should solve this in this particular PR? It matches the name of the internal texture so that's pretty much why I used that.

Best alternatives I can think of is something like Color Buffer or related.